client/src/assets/scss/_new.scss

```diff
@@ -2367,6 +2367,13 @@ div.page-activity.prenatal {
             color: $color-green;
           }
         }
+
+        .nutritional-supplement-alert {
+          color: $color-red-alert;
```

Comment:
Do we have `color-red` and `color-red-alert` separately?

Reply:
@AronNovak  Yes
![Selection_1424](https://github.com/user-attachments/assets/50ccfc8d-72a0-401b-b06e-2efa9c05e6eb)